doc: add descriptions for helm values. (#832)
longhorn/website: content/docs/1.6.0/references/helm-values.md

| annotations | `{}` | Annotations to add to the Longhorn Manager DaemonSet Pods. Optional. |
| enableGoCoverDir | `false` | Enable this to allow Longhorn to generate code coverage profiles |
| enablePSP | `false` | For Kubernetes < v1.25, if your cluster enables Pod Security Policy admission controller, set this to `true` to ship longhorn-psp which allow privileged Longhorn pods to start |

> | helmPreUpgradeCheckerJob.enabled | `true` | Enable this to allow Longhorn to do pre-upgrade checks. Disable this to support ArgoCD. |

Reviewer: ```suggestion
| helmPreUpgradeCheckerJob.enabled | `true` | Setting that allows Longhorn to perform pre-upgrade checks. Disable this setting when installing Longhorn using Argo CD. |
```

Reviewer: Note: The official spelling of [Argo CD](https://argoproj.github.io/cd/) has a space.

Reviewer 2: >Setting that allows Longhorn to perform pre-upgrade checks. Disable this setting when installing Longhorn using Argo CD or other GitOps solutions. 

@mantissahz @jillian-maroket

Author: Modified. Thanks.

> | metrics.serviceMonitor.enabled | `false` | Enable this to create the ServiceMonitor |

Reviewer: ```suggestion
| metrics.serviceMonitor.enabled | `false` | Setting that allows the creation of ServiceMonitor. |
```

Reviewer 2: Please also update https://longhorn.io/docs/1.6.0/monitoring/prometheus-and-grafana-setup/#install-longhorn-servicemonitor to make it optional if users deploy Longhorn with this setting enabled.

Reviewer 2: > Setting that allows the creation of a Prometheus ServiceMonitor resource for longhorn-manager components. 

@jillian-maroket @mantissahz
